docs/modules/operation/pages/deep-dive/topology/enlinkd/introduction.adoc

```diff
@@ -234,8 +234,12 @@ NOTE: If multiple protocols are enabled, the links will be discovered for each e
 | false
 |===
 
+**Warning: disable discovery protocol delete related persisted data.
+[.line-through]#**#
+
 The discovery for bridge first start is scheduled at `initial_sleep_time + bridge_topology_interval`.
-The updaters' first start is scheduled at 0L.
+[.line-through]#The# updaters' first start is scheduled at 0L.
```

Comment:
I'm not clear on why we are using the ".line-through" syntax.

Reply:
I'm sorry I'm not confident with docs. I just want this information on a new line.
If you propose style changes I'll be so glad.

Reply:
No worries at all. I wasn't sure if I was missing something. You just need to add a line between. I'll update.

```suggestion

The updaters' first start is scheduled at 0L.
```